README.md

```diff
@@ -7,19 +7,19 @@ View the templates at https://openshift.github.io/oauth-templates/.
 ## Development
 
 1. Install [Jekyll](https://jekyllrb.com/docs/installation/).
-1. Run `bundle install`.
-1. Run `bundle exec jekyll serve`.
+2. Run `bundle install`.
```

Comment:
Nit: here and other unordered list numbering--keeping them all starting with `1.` means no renumbering is ever required.